templates/dna/replace.js

```diff
@@ -0,0 +1,29 @@
+var fs = require('fs')
+var path = require('path')
+var templateC = 'Note'
+var template = 'note'
+var zome = process.argv[2].toLowerCase()
+var zomeC = zome.charAt(0).toUpperCase() + zome.substring(1)
+
+function replace(fileName, template, zome) {
+  var templateFile = fs.readFileSync(path.resolve(__dirname, fileName)).toString()
+  newContent = templateFile.replace(new RegExp(template, 'g'), zome)
+  fs.writeFileSync(path.resolve(__dirname, fileName), newContent)
+}
+
+replace('zomes/'  + zome + 's/zome.json', template, zome)
+replace('zomes/'  + zome + 's/code/.hcbuild', template, zome)
+replace('zomes/'  + zome + 's/code/cargo.toml', template, zome)
+replace('zomes/'  + zome + 's/code/src/lib.rs', templateC, zomeC)
+replace('zomes/'  + zome + 's/code/src/lib.rs', template, zome)
+fs.renameSync(path.resolve(__dirname, 'zomes/'  + zome + 's/code/src/' + template), path.resolve(__dirname, 'zomes/'  + zome + 's/code/src/' + zome))
+replace('zomes/'  + zome + 's/code/src/'  + zome + '/mod.rs', templateC, zomeC)
+replace('zomes/'  + zome + 's/code/src/'  + zome + '/mod.rs', template, zome)
+replace('zomes/'  + zome + 's/code/src/'  + zome + '/mod.rs', template.toUpperCase(), zome.toUpperCase())
```

Comment:
not super important, but you could write `replace` to take an object, and then just make one call per file. This would shorten the code in this file and mean fewer disk reads and writes.

something like
```Javascript
replace('zomes/'  + zome + 's/code/src/'  + zome + '/mod.rs', {
  [templateC]: zomeC,
  [template]: zome,
  [template.toUpperCase()]: zome.toUpperCase()
})
```
or just an array of pairs, which is a little easier to digest in the function
`[[templateC, zomeC], [template, zome], [template.toUpperCase(), zome.toUpperCase()]]`